### PR TITLE
Add medical-record pallet to runtime, test functionality on polkadot.js

### PR DESCRIPTION
**Pull request by OverOrion**
This PR integrates the newly created pallet into runtime, so it can be used via polkadot.js

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -255,6 +258,17 @@ impl pallet_balances::Config for Runtime {
 	type WeightInfo = pallet_balances::weights::SubstrateWeight<Runtime>;
 }
 
+type MaxRecordContentLength = ConstU32<300>;
+type SignatureLength = ConstU32<150>;
```
Comment (LRUki) [inline runtime/src/lib.rs:262]: I guess we will change this depending on the encryption method we use later on?

**Review by LRUki — APPROVED** [review]
LGTM! GJ @OverOrion @PieWol

File: runtime/src/lib.rs (hunk 1 above)
Comment (OverOrion) [reply runtime/src/lib.rs:262]: Yep, these numbers are placeholders at the moment.

Comment (OverOrion) [reply runtime/src/lib.rs:262]: Yep, these numbers are placeholders at the moment.

**Pull request merged**